Dockerfile

```diff
@@ -4,7 +4,7 @@
 # /make/builder.Dockerfile
 # /taprpc/Dockerfile
 # /tools/Dockerfile
-FROM golang:1.20.5-alpine as builder
+FROM golang:1.21.0-bookworm as builder
```

Comment:
We can't just switch from `-alpine` (completely different Linux distro with a different package manager) over to Debian. And the Alpine ones are more stable in terms of us just being able to bump the version numbers, compared to Debian. So I'd rather do it the other way, switch from Debian to Alpine (but that requires deeper changes as well, so maybe let's defer that).

Reply:
This switch of distro is still in here?

Reply:
Oh yes, I missed this one in my review.